docs/PrismaMigrationTools.md

```diff
@@ -0,0 +1,64 @@
+# Prisma Migration Tools
+
+## `npm prisma:seed`
+
+### What it does
+
+This command seeds the database, which means populating the database with initial data it requires to run.
+It can be dummy data, basic user accounts, a default language, etc.
+
+### Why to use it
+
+While developing the application, we need some data to be present in order for something to appear on the front end to interact with.
+Running this command populates the database with a basic set of data for development purposes.
+
+For more detailed information about [prisma seed](https://www.prisma.io/docs/guides/database/seed-database), click the link.
+
+## `npm prisma:reset`
+
+### What it does
+
+This command drops the database if possible or performs a soft reset if the database cannot be deleted.
+If the database is dropped, it creates a new database with the same name and runs all migrations and seed scripts.
+If `npm prisma:reset` runs without any errors, then the `npm prisma:seed` function is auto-invoked.
+This means you do not need to run the `npm prisma:seed` command again.
+
+### Why to use it
+
+If you need to undo changes you made to the database schema or if there are migration history conflicts, you should run this command.
+
+You use this command to also restore the data to its original state if you sufficiently modified the data via the front end.
```

Comment:
let's reverse these two lines. restoring the data is probably the primary reason for devs running this command, and the other sentence describes the potential alternative reasons. what do you think?